src/asciidoc/docs/guides-general/capture-user-input.adoc

```diff
@@ -0,0 +1,139 @@
+= Capturing user input
+:page-slug: capture-user-input
+:page-description: How to capture user input using `Koji.ui.capture` controls.
+:includespath: ../_includes
+:imagesdir: path-to-images-directory
+
+Koji provides a set of predefined user-input controls that you can use in your app's frontend to capture information and content from your users.
+
+These controls are found in `Koji.ui.capture` in the @withkoji/core package.
+
+== Using user-input controls
+
+To use a predefined user-input control in your app,
+
+* Install @withkoji/core in the *frontend* terminal tab.
++
+.Learn how...
+[%collapsible]
+====
+include::{includespath}/steps-codeeditor.adoc[tag=cli-front-end]
+include::{includespath}/steps-codeeditor.adoc[tag=terminal-stop-process]
+include::{includespath}/steps-codeeditor.adoc[tag=terminal-install-withkoji-core-package]
+include::{includespath}/steps-codeeditor.adoc[tag=terminal-restart-process]
+====
+
+* Import @withkoji/core in your front-end code.
++
+[source,javascript]
+----
+import Koji from '@withkoji/core';
+----
+
+* Call the appropriate `Koji.ui.capture` control.
++
+[source,javascript]
+----
+myVariable = await Koji.ui.capture.<control-name>( <control-parameters> )
+----
++
+** `.audio` – Prompts the user to select or upload an audio file.
+<<core-frontend-ui-capture#audio,API reference>>
+
+** `.color` – Prompts the user to select or specify a color.
+<<core-frontend-ui-capture#color,API reference>>
++
+If `allowAlpha : true` is included in the options, the *Opacity* slider is visible in the color selector modal.
+
+** `.custom` – Prompts the user to interact with a custom control.
+<<core-frontend-ui-capture#custom,API reference>> +
+Learn more: <<build-custom-controls#>>
+
+** `.file` – Prompts the user to upload a file of any type.
+<<core-frontend-ui-capture#file,API reference>>
+
+** `.image` – Prompts the user to select or upload an image file.
+<<core-frontend-ui-capture#image,API reference>>
+
+** `.link` – Prompts the user to enter an external URL.
+<<core-frontend-ui-capture#link,API reference>>
+
+** `.media` – Prompts the user to select, specify, or upload an image, a file, an audio, or a video.
+<<core-frontend-ui-capture#media,API reference>>
+
+** `.range` – Prompts the user to select a value from a numeric range.
+<<core-frontend-ui-capture#range,API reference>>
+
+** `.select` – Prompts the user to select from a predefined list of options.
+<<core-frontend-ui-capture#select,API reference>>
++
+If the list has 1 to 4 options, the options are presented as radio buttons. +
+If the list has 5 or more options, the list is presented as a dropdown.
+
+** `.video` – Prompts the user to select or upload a video file.
+<<core-frontend-ui-capture#video,API reference>>
+
+== Controls for various file types
+
+The @withkoji/core package provides multiple APIs for selecting or uploading files.
```

Comment:
Change "API" to "method" throughout. These are all methods that are part of the same API.

Reply:
Fixed.